test/unit/fh3/resources/test_resources.js

```diff
@@ -37,5 +40,12 @@ module.exports = {
       assert.equal(data.state,"RUNNING");
       return cb();
     });
+  },
+  'fhc resources list --env': function(cb) {
+    listResourcesCmd({env:'dev'}, function(err) {
+      assert.equal(err, null);
```

Comment:
Maybe also check `dataSet` is valid? `assert.equal(dataSet.thing, 'thingvalue')`?